lib/danger/ci_source/gitlab_ci.rb

```diff
@@ -54,16 +61,29 @@ def self.determine_merge_request_id(env)
 
     def initialize(env)
       @env = env
-      @repo_slug = env["CI_MERGE_REQUEST_PROJECT_PATH"] || env["CI_PROJECT_PATH"]
-      @project_url = env["CI_MERGE_REQUEST_PROJECT_URL"] || env["CI_PROJECT_URL"]
+      @repo_slug = slug_from(env)
+      @project_url = env["DANGER_PROJECT_REPO_URL"] || env["CI_MERGE_REQUEST_PROJECT_URL"] || env["CI_PROJECT_URL"]
```

Comment:
On an unrelated note, there is a separate PR #1237 to remove this `project_url` as it does not seem to be used anymore.